examples/specs/window_transform_mean_difference_by_year.vl.json

```diff
@@ -0,0 +1,45 @@
+{
+    "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
+    "description": "Bar graph showing the best films for the year they were produced, where best is defined by at least 2.5 points above average for that year.",
+    "width": 300,
+    "height": 50,
+    "data": {
+        "url": "data/movies.json",
+        "format": {
+            "parse": { "Release_Date": "date:'%d-%b-%y'" }
+        }
+    },
+    "transform": [
+        { "timeUnit": "year", "field": "Release_Date", "as": "year" },
+        {
+            "window": [{
+                "op": "mean",
+                "field": "IMDB_Rating",
+                "as": "AverageYearRating"
+            }],
+            "groupby": [
+                "year"
+            ],
+            "frame": [null, null]
+        },
+        {
+            "calculate": "datum.IMDB_Rating - datum.AverageYearRating",
+            "as": "RatingDelta"
+        },
+        {
+            "filter": "datum.RatingDelta > 2.5"
+        }
+    ],
+    "mark": { "type": "bar", "clip": true },
+    "encoding": {
+        "x": {
```

Comment:
Try either transposing the plot (swap x and y) or adjust x's axis labelAngle so it's easier to read.  (Same for the other plot.)